frontend/packages/integration-tests-cypress/plugins/index.js

```diff
@@ -48,6 +48,12 @@ module.exports = (on, config) => {
   config.baseUrl = `${process.env.BRIDGE_BASE_ADDRESS || 'http://localhost:9000'}${(
     process.env.BRIDGE_BASE_PATH || '/'
   ).replace(/\/$/, '')}`;
+  config.env.BRIDGE_HTPASSWD_IDP = process.env.BRIDGE_HTPASSWD_IDP;
+  config.env.BRIDGE_HTPASSWD_USERNAME = process.env.BRIDGE_HTPASSWD_USERNAME;
+  config.env.BRIDGE_HTPASSWD_PASSWORD = process.env.BRIDGE_HTPASSWD_PASSWORD;
   config.env.BRIDGE_KUBEADMIN_PASSWORD = process.env.BRIDGE_KUBEADMIN_PASSWORD;
+  config.env.appHost = `${process.env.BRIDGE_BASE_ADDRESS || 'http://localhost:9000'}${(
```

Comment:
This looks like it's the same as `baseUrl`. Can we have one config value tracking?

Reply:
wow, talk about cut & paste blinders :-)  sorry, removed `appHost`.